ui/app/templates/components/oidc/client-form.hbs

```diff
@@ -37,7 +37,7 @@
   </div>
   {{! RADIO CARD + SEARCH SELECT }}
   <div class="box is-sideless is-fullwidth is-marginless has-top-padding-xxl">
-    <h4 class="title is-4">Assign access</h4>
+    <h2 class="title is-4">Assign access</h2>
```

Comment:
I'm not sure if discussed this, but should we use the HDS text component? https://helios.hashicorp.design/components/text

Reply:
Sure, I can do that!